db/fdb_fend.c

```diff
@@ -4349,8 +4349,23 @@ int fdb_trans_commit(sqlclntstate *clnt, enum trans_clntcomm sideeffects)
             if (tran->nwrites) {
                 /* handle is only created upon first remote write to this fdb */
                 assert(tran->fcon.hndl);
+
                 fdb_client_set_identityBlob(clnt, tran->fcon.hndl);
                 rc = cdb2_run_statement(tran->fcon.hndl, "commit");
+                if (!rc && !clnt->verifyretry_off) {
```

Comment:
Oh I guess if clnt verify retry is off then we already store effects within transaction so don't need to do here

Reply:
Although there could potentially be cases where effects right before commit are different than effects after commit